test/config/utility.cc

```diff
@@ -822,7 +822,8 @@ void ConfigHelper::addClusterFilterMetadata(absl::string_view metadata_yaml,
 
 void ConfigHelper::setConnectConfig(
     envoy::extensions::filters::network::http_connection_manager::v3::HttpConnectionManager& hcm,
-    bool terminate_connect, bool allow_post, bool http3) {
+    bool terminate_connect, bool allow_post, bool http3,
+    absl::optional<envoy::config::core::v3::ProxyProtocolConfig::Version> proxy_protocol_version) {
```

Comment:
nit: OptRef

Reply:
as discussed out of band ref would be larger than underlying enum int32_t